### PR TITLE
expand reviewer guidelines in pull request template

### PR DESCRIPTION
**Pull request by grlee77**
## Description

Adds helpful reminders for how to run benchmarks and how to request an automated backport of a PR to another branch. @hmaarrfk, I think you had requested this recently for the backport command? I agree that I often forget the exact notation as well and have to look it up on a prior PR! It seemed the benchmarking feature might benefit from a reminder as well.

## Checklist

- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- Gallery example in `./doc/examples` (new features only)
- Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- Unit tests
- Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
- Descriptive commit messages (see below)

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -27,4 +27,10 @@
   later.
 - Check that new functions are imported in corresponding `__init__.py`.
 - Check that new features, API changes, and deprecations are mentioned in
-      `doc/release/release_dev.rst`.
+    `doc/release/release_dev.rst`.
```
Comment (mkcor) [inline .github/PULL_REQUEST_TEMPLATE.md:30]:
```suggestion
  `doc/release/release_dev.rst`.
```
(see "later" line 27)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -27,4 +27,10 @@
...
+- There is a bot to help automate backporting a PR to an older branch. For
+    example, to backport to v0.19.x after merging, add the following in a PR
+    comment: `@meeseeksdev backport to v0.19.x`
```
Comment (mkcor) [inline .github/PULL_REQUEST_TEMPLATE.md:33]:
```suggestion
  example, to backport to v0.19.x after merging, add the following in a PR
  comment: `@meeseeksdev backport to v0.19.x`
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -27,4 +27,10 @@
...
+- To run benchmarks on a PR, add the `run-benchmark` label. To rerun, the label
+    can be removed and then added again. The benchmark output can be checked in
+    the "Actions" tab.
```
Comment (mkcor) [inline .github/PULL_REQUEST_TEMPLATE.md:36]:
```suggestion
  can be removed and then added again. The benchmark output can be checked in
  the "Actions" tab.
```

**Review by mkcor — APPROVED** [review]
Nice!

**Review by hmaarrfk — APPROVED** [review]

**Comment by grlee77** [comment]
> Seems like the bot got triggered hahah

If that happens on all new PRs due to it being in the template, then that is a bit of a problem. I guess we could list it with a space between `@` and `meeseeksdev` if necessary. Hopefully the backtics will prevent that, though.

**Comment by hmaarrfk** [comment]
good point. let's see what happens.

**Pull request merged**